static/js/opening.js

```diff
@@ -0,0 +1,8 @@
+// let splitText = new SplitText('h1', {type: 'chars'});
```

Comment:
This file can be deleted from this branch. This is part of the same backlog ticket I mentioned above.

Reply:
This file is deleted.